src/ViewerComponents/Image/ImagesContainer.svelte

```diff
@@ -112,36 +142,53 @@
      * @returns {boolean}
      */
     function checkStillImage(data) {
-        if (data && Array.isArray(data)) {
-            return data.every(obj => obj['knora-api:hasStillImageFileValue']);
+        if (data.hasOwnProperty('@graph') && Array.isArray(data['@graph'])) {
+            return data['@graph'].every(obj => obj['knora-api:hasStillImageFileValue']);
         } else {
-            return false;
+            return data.hasOwnProperty('knora-api:hasStillImageFileValue');
         }
     }
 
+    /**
+     * Wraps the data in an array if there is one or no result.
+     *
+     * @param data
+     */
+    function wrapData(data) {
```

Comment:
Maybe this function should be in a service as we are going to use it quiet often

Reply:
Will be extracted